libnixt/include/nixt/Deserialize.h

```diff
@@ -3,8 +3,8 @@
 #include "HookExpr.h"
 #include "PtrPool.h"
 
+#include <nix/expr/nixexpr.hh>
 #include <nix/input-accessor.hh>
```

Comment:
`input-accessor.hh` doesn't exist anymore, yet I built everything. Is this file dead code?

Reply:
Yes. Related issue: #574